w09h01/test/pgdp/pingutrip/LegalCheck.java

```diff
@@ -0,0 +1,261 @@
+// ************* PLEASE READ THIS DISCLAIMER ************* //
+/*                                                          *
+ * Even when there is a test provided by Artemis to test    *
+ * the usage of not allowed things, you could still lose    *
+ * points after the deadline, if you are using not allowed  *
+ * things. See the link below                               *
+ *                                                          *
+ * https://zulip.in.tum.de/#narrow/stream/1503-PGdP-W09H01  *
+ * /topic/Nach.20Deadline.20Abzug.20von.20Punkten/near      *
+ * /861199                                                  *
+ *                                                          *
+ * THIS TEST CHECKS FOR INSTANCES OF KEYWORDS WITHIN YOUR   *
+ * CODE. IN THE CASE OF FALSE POSITIVE UNCOMMENT LINE 39    *
```

Comment:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/24459435/208238762-0ee393de-5077-4585-a5f5-8fcdb8b18552.png">

Isn't it line 38?